src/pysdmx/model/message.py

```diff
@@ -182,3 +187,15 @@ def get_dataset(self, short_urn: str) -> Dataset:
             f"No Dataset with Short URN {short_urn} found in data.",
             "Could not find the requested Dataset.",
         )
+
+    def get_transformation_schemes(self) -> List[TransformationScheme]:
+        """Returns the TransformationSchemes."""
+        return self.__get_elements(TransformationScheme)
+
+    def get_udo_schemes(self) -> List[UserDefinedOperatorScheme]:
```

Comment:
I'm not active in the VTL space but is `udo` a known acronym there? If not, maybe we should rename the function to something like `get_operator_schemes` or similar?